images/installer/Dockerfile

```diff
@@ -1,7 +1,7 @@
 # Using playbook2image as a base
 # See https://github.com/openshift/playbook2image for details on the image
 # including documentation for the settings/env vars referenced below
-FROM registry.centos.org/openshift/playbook2image:latest
+FROM openshift/base-centos7
```

Comment:
uh... why this one? This brings in a [bunch of dev stuff](https://hub.docker.com/r/openshift/base-centos7/) we don't need. Why not just `centos7`?